pom.xml

```diff
@@ -128,6 +128,16 @@
         <artifactId>gemini-resin-archetype</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>gemini-jdbc</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>gemini-hikaricp</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Comment:
Do these entries need to be added to the `modules` section near the top of this `pom.xml`?